index.html

```diff
@@ -0,0 +1,233 @@
+                    <li class="tag">HTML</li>
+                    <li class="tag">CSS</li>
+                    <li class="tag">JavaScript</li>
+                </ul>
+                <div class="projectDemo">
+                    <button class="enabled">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    
+    <section id="aboutme">
+        <div class="top">
+            <span class="heading">About Myself</span>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="letconnect" href="">LET’S CONNECT</a> 
+            <ul class="socialIconsGrid">
+                <li><a href="#"><img src="./assets/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="./assets/linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="./assets/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="./assets/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
+            </ul>
+            <div class="action"><button class="enabled">Get my resume</button></div>
+        </div>
+
+        <div class="myskills">
+            <div class="languages">
+                <div class="lan_head">
+                    <div class="lan_heading">
+                        <h5>Languages</h5>
+                    </div>
+                    <a class="dropdown_icon">
+                        <span class="material-symbols-outlined">
+                            expand_more
+                    </span>
+                    </a>
+					</div>
+            <ul class="lan_list">
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/javascript_icon.svg" alt="javascript">
+									<p>Javascript</p>
+							</li>
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/html_icon.svg" alt="html">
+								<p>HTML</p></li>
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/css_icon.svg" alt="css">
+								<p>CSS</p></li>
+						</ul>
+				</div>
+				<div class="languages">
+					<div class="lan_head">
+						<div class="lan_heading">
+							<h5>Frameworks</h5>
+						</div>
+						<a class="dropdown_icon">
+							<span class="material-symbols-outlined">
+								expand_more
+								</span>
+						</a>
+					</div>
+				</div>
+                <hr/>
+				<div class="languages">
+					<div class="lan_head">
+						<div class="lan_heading">
+							<h5>Skills</h5>
+						</div>
+						<a class="dropdown_icon">
+							<span class="material-symbols-outlined">
+								expand_more
+								</span>
+						</a>
+					</div>
+				</div>
+                <hr/>
+			</div>
+    </section>
```

Comment:
This section should have proper indentation

Reply:
- [ ] Great effort using Github inline features to perform the review. Please make sure the review meets each of the following.
The comments should be polite. You can offer them as suggestions instead of commands 👍🏾 
Include an explanation as to why the change is necessary 👍🏾 
The comment should be clear and easy to understand 👍🏾